includes/definitions/alpineoe-tdcmedfa.yaml

```diff
@@ -0,0 +1,16 @@
+os: alpineoe-tdcmedfa
+text: 'Alpine OptoElectronics TDCM-EDFA platform'
+type: appliance
+icon: alpineoe
+mib_dir: alpineoe
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - 
+        sysObjectID:
+            - .1.3.6.1.4.1.52326.
+    -
+        sysDescr:
+            - One-U System
```

Comment:
is this needed? isnt the sysobjectid check enough?

Reply:
Unfortunately not, when we remove the sysDescr field, the device is discovered as "Generic Device"

Reply:
because of the dot in the end I guess

Reply:
That's right, I'm going to correct this.

Reply:
That's right, I'm going to remove the sysDescr field.